### PR TITLE
Topic 4 Exercises 1 and 2

### PR DESCRIPTION
**Pull request by famecrew**

### CODE REVIEW

File: 04-HTML5-APIs/assets/scripts/script.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+// Local storage
+const localStorageText = document.getElementById("localStorageText");
+document.onload = recoverText();
+
+function localStorageSave() {
+    window.localStorage.setItem("textArea", localStorageText.value);
```
Comment (DemianMoschin-GLB) [inline 04-HTML5-APIs/assets/scripts/script.js]: Great! 

Please keep string comments with single quotes, and remain with this convention across your app.

Comment (famecrew) [reply 04-HTML5-APIs/assets/scripts/script.js]: I love double quotes, I'll change it

Comment (famecrew) [reply 04-HTML5-APIs/assets/scripts/script.js]: Done!

File: 04-HTML5-APIs/assets/scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
+// Local storage
+const localStorageText = document.getElementById('localStorageText');
+document.onload = recoverText();
+
+function localStorageSave() {
+    window.localStorage.setItem('textArea', localStorageText.value);
```
Comment (DemianMoschin-GLB) [inline 04-HTML5-APIs/assets/scripts/script.js]: is the window word necessary ? if not, please remove

**Comment by DemianMoschin-GLB** [comment]
Great work!

File: 04-HTML5-APIs/assets/scripts/script.js (hunk 2 above)
Comment (famecrew) [reply 04-HTML5-APIs/assets/scripts/script.js]: Done, removed where it is not necessary

**Pull request closed without merging**